apps/api/src/tasks/task.entity.ts

```diff
@@ -74,6 +74,9 @@ export class TaskEntity implements Task {
   @Column({ name: 'ended_at', nullable: true, type: 'timestamptz' })
   endedAt: Date;
 
+  @Column({ type: 'simple-array', default: '' })
```

Comment:
why not nullable like the other one ?

Reply:
want to avoid checks if array exists. This makes code simple. Empty array represented by empty string, should not make any difference for db.

Reply:
its more from an API perspective, to match the client side... having to supply an empty array for all tasks